docs/python/reference/client.md

```diff
@@ -0,0 +1,225 @@
+---
+sidebar_position: 10
+title: DBOS Client
+---
+
+`DBOSClient` provides a programmatic way to interact with your DBOS application from external code.
+`DBOSClient` includes methods similar to [`DBOS`](./contexts.md)
+that make sense to be used outside of a DBOS workflow or step, such as 
+[`enqueue`](./queues.md#enqueue) or [`getEvent`](./contexts.md#get_event).
```

Comment:
I think it is unclear what "make sense" means. We can just delete this sentence.

```suggestion
```

Reply:
I don't think deleting it is a good idea. Will reword